en/extend_your_application/README.md

```diff
@@ -37,7 +37,7 @@ We will start with adding a link inside `blog/templates/blog/post_list.html` fil
 
 {% raw %}Time to explain the mysterious `{% url 'post_detail' pk=post.pk %}`. As you might suspect, the `{% %}` notation means that we are using Django template tags. This time we will use one that will create a URL for us!{% endraw %}
 
-`blog.views.post_detail` is a path to a `post_detail` *view* we want to create. Please note: `blog` is the name of our application (the directory `blog`), `views` is from the name of the `views.py` file and the last bit – `post_detail` – is the name of the *view*.
+`post_detail` is a path to a `post_detail` *view* we want to create. Please note: `blog` is the name of our application (the directory `blog`), `views` is from the name of the `views.py` file and the last bit – `post_detail` – is the name of the *view*.
```

Comment:
@olasitarska now I rebase and change only `django_forms` READMEs, but in `extend_your_application` READMEs `blog.views.post_detail` it's not correct because in sample code it's not used. In this PR I fix the error in `django_forms`, then I can look at the other problem in another PR ;)